src/repositories/user.repository.ts

```diff
@@ -105,4 +105,12 @@ export const findUserByUsername = async (
   }
 };
 
+export const changeStatsById = (
+  id: string,
+  fieldName: string,
+  value: number
+) => {
+  return UserModel.updateOne({ _id: id }, { $inc: { [fieldName]: value } });
```

Comment:
```suggestion
  return UserModel.findOneAndUpdate({ _id: id }, { $inc: { [fieldName]: value } },{
  new: true
});
```
https://mongoosejs.com/docs/tutorials/findoneandupdate.html